pkg/datapath/iptables/ipset/ops.go

```diff
@@ -30,37 +31,61 @@ type ops struct {
 	ipset   *ipset
 }
 
-func (ops *ops) Update(ctx context.Context, _ statedb.ReadTxn, entry *tables.IPSetEntry, changed *bool) error {
+// DeleteBatch implements reconciler.BatchOperations.
+func (ops *ops) DeleteBatch(ctx context.Context, txn statedb.ReadTxn, batch []reconciler.BatchEntry[*tables.IPSetEntry]) {
```

Comment:
Nit. It seems a bit more natural if UpdateBatch comes before DeleteBatch (and the diff would also improve a bit).